packages/itwinui-react/src/core/Stepper/StepperStep.tsx

```diff
@@ -37,6 +37,22 @@ export type StepperStepProps = {
    * A tooltip giving detailed description to this step.
    */
   description?: string;
+  /**
+   * Allows props to be passed for iui-stepper-step
```

Comment:
Make sure the class name is removed from these jsdocs as well

Reply:
All of the new props in this file also reference the class name. Make sure they're all changed

Reply:
fixed